github/http.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/github/hub/utils"
 )
 
-const apiPayloadVersion = "application/vnd.github.v3+json;charset=utf-8"
+const apiPayloadVersion = "application/vnd.github.black-cat-preview+json"
```

Comment:
Could we only specify this header value for the `POST /repos/%s/%s/pulls/%d/requested_reviewers` request instead for all requests?